environments/common.yml

```diff
@@ -0,0 +1,19 @@
+name: sklearn-env
+
+# These should be 1-1 with requirements/common.txt
+dependencies:
+    - wheel
+    - python=3.9
```

Comment:
should we use 3.10?

Reply:
The live documentation asks for `3.9` but if we have bumped dependencies and the docs are outdated then that is fine and I can update. Otherwise, we could leave to a follow-up PR.

Reply:
I think that we should change that. We should only have an up-to-date upper bound.